rest-api/src/main/java/gov/cms/qpp/conversion/api/config/DynamoDbConfig.java

```diff
@@ -35,12 +34,21 @@
 	private static final Logger API_LOG = LoggerFactory.getLogger(Constants.API_LOG);
 	static final String NO_KMS_KEY = "No KMS key specified!";
 
-	@Autowired
 	private Environment environment;
-
-	@Autowired
 	private AWSKMS awsKms;
 
+
+	/**
+	 * Ensure required dependencies are supplied.
+	 *
+	 * @param environment access to environment variables
+	 * @param awsKms KMS key
+	 */
+	public DynamoDbConfig(final Environment environment, final AWSKMS awsKms) {
```

Comment:
Do you not need to add an @Inject to the constructor?

Reply:
Not necessarily. We could add them to be more explicit, but since Spring 4.3 beans with a single constructor are injected without needing annotation.